mix.exs

```diff
@@ -237,12 +236,14 @@ defmodule Trento.MixProject do
     ]
   end
 
-  defp get_version, do: System.get_env("VERSION", get_version_from_git())
-
-  defp get_version_from_git do
-    case File.cwd!() |> Path.join("hack/get_version_from_git.sh") |> System.cmd([]) do
-      {version, 0} -> version |> String.trim("\n")
-      _ -> @version
+  defp get_version do
+    case System.get_env("VERSION") do
```

Comment:
Be careful here: `System.get_env("VERSION")` returns `nil` if the env var `VERSION` is not found.

I guess we actually need `System.get_env("VERSION", "")` which returns an empty string if `VERSION` is not defined as en env var, effectively covering all cases.

Reply:
ohh nice catch.